client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/EmbeddedBackfila.kt

```diff
@@ -35,6 +37,30 @@ internal class EmbeddedBackfila @Inject internal constructor(
     return Calls.response(ConfigureServiceResponse())
   }
 
+  override fun createAndStartbackfill(request: CreateAndStartBackfillRequest): Call<CreateAndStartBackfillResponse> {
+    checkNotNull(serviceData) { "Must register the service before creating a backfill" }
+
+    val createRequest = request.create_request
+    checkNotNull(serviceData!!.backfills.firstOrNull { it.name == createRequest.backfill_name }) {
+      "Backfill ${createRequest.backfill_name} was not registered properly"
+    }
+
+    val backfillId = backfillIdGenerator.getAndIncrement()
+    val operator = operatorFactory.create(createRequest.backfill_name, backfillId.toString())
+
+    val run = EmbeddedBackfillRun<Backfill<*, *>>(
+        operator = operator,
+        dryRun = createRequest.dry_run,
+        parameters = createRequest.parameter_map,
+        rangeStart = createRequest.pkey_range_start.utf8(),
+        rangeEnd = createRequest.pkey_range_end.utf8()
+    )
+
+    run.execute()
```

Comment:
This I think actually runs the whole thing if I remember correctly. Do we want that? I'm fine if the answer is yes

Reply:
yeah I want that, makes the most sense for testing